pkg/cmd/bootstrap.go

```diff
@@ -154,11 +159,11 @@ func missingFlagErr(flags []string) error {
 }
 
 // initiateInteractiveMode starts the interactive mode impplementation if no flags are passed.
-func initiateInteractiveMode(io *BootstrapParameters) error {
+func initiateInteractiveMode(io *BootstrapParameters, client *utility.Client) error {
 	log.Progressf("\nStarting interactive prompt\n")
 	// ask for sealed secrets only when default is absent
-	if io.SealedSecretsService == (types.NamespacedName{}) {
-		io.SealedSecretsService.Name = ui.EnterSealedSecretService(&io.SealedSecretsService)
+	if client.CheckIfSealedSecretsExists(defaultSealedSecretsServiceName) != nil {
+		io.SealedSecretsService.Namespace = ui.EnterSealedSecretNamespace(&io.SealedSecretsService)
```

Comment:
EnterSealedSecretNamespace is a misnomer.  namesspace and name entered by the user.